assert/assertions_test.go

```diff
@@ -788,6 +788,27 @@ func TestElementsMatch(t *testing.T) {
 	}
 }
 
+func TestDiffLists(t *testing.T) {
+	check := func(listA, listB interface{}, extraA, extraB []interface{}) {
```

Comment:
Thank you, one last thing.

Would it not be clearer if we convert these unit tests to table-driven tests? (https://github.com/golang/go/wiki/TableDrivenTests)

Reply:
Okay, updated.